src/cljs/main/broadfcui/components/ontology_autosuggest.cljs

```diff
@@ -1,37 +1,64 @@
 (ns broadfcui.components.ontology-autosuggest
   (:require
    [dmohs.react :as react]
+   [clojure.string :as string]
+   [broadfcui.common.flex-utils :as flex]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.links :as links]
+   [broadfcui.common.style :as style]
    [broadfcui.components.autosuggest :refer [Autosuggest]]
    [broadfcui.utils :as utils]
    ))
 
+(defn render-doid [doid]
+  (-> doid
+      (string/split "/")
+      last
+      (string/replace-all #"_" ": ")))
 
-(react/defc OntologyAutosuggest
```

Comment:
was this explicitly un-componentized for a reason?

Reply:
yes, isaac suggested it. it was extra overhead when we just needed to customize some properties